managing_providers/_topics/adding_an_azure_kubernetes_provider.md

```diff
@@ -0,0 +1,71 @@
+# Adding an Azure Kubernetes Service (AKS) Container Provider
+
+After initial installation and creation of a {{ site.data.product.title_short }}
+environment you can add an Azure Kubernetes Service Container provider.
```

Comment:
should the above 2 lines be one-lined?

Reply:
Pulled this from https://raw.githubusercontent.com/ManageIQ/manageiq-documentation/master/managing_providers/_topics/adding_an_openshift_provider.md it is one-line in rendered markdown.